HW_3/templates/register.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Регистрация</title>
+  </head>
+  <body>
+  <div align="center">
+    <h1>Регистрация</h1>
+    <form method="POST" action="{{ url_for('register') }}">
+      {{ form.hidden_tag() }}
+      <p>
+        {{ form.first_name.label }}<br>
+        {{ form.first_name(size=32) }}
+      </p>
+      <p>
+        {{ form.last_name.label }}<br>
+        {{ form.last_name(size=32) }}
+      </p>
+      <p>
+        {{ form.email.label }}<br>
+        {{ form.email(size=32) }}
+      </p>
+      <p>
+        {{ form.password.label }}<br>
+        {{ form.password(size=32) }}
+      </p>
+      <p>{{ form.submit() }}</p>
```

Comment:
Приводим код по принципу DRY
```suggestion
      {{ form.csrf_token }}
      {% for field in form if field.name != 'csrf_token' %}
      <p>
        {{ field.label }}<br>
        {{ field(size=32) }}
      </p>
      {% endfor %}
Намного лучше, не так ли?